doc/api/index.apib

```diff
@@ -0,0 +1,45 @@
+FORMAT: 1A
+# Target API
+
+# Group Topics
+
+## Index topics [/api/v1/topics]
+
+### Index [GET]
+
++ Request Index - OK - Without params
+
+    + Headers
+
+            Access-Token: H5DVUqDOCXK8eRD652Jgpw
```

Comment:
This data should be replaced with the one you hardcoded
[spec/support/devise_token_auth_overrides.rb](https://github.com/LeandroSoria2/TargetApi/pull/21/files#diff-e55b07c29170e3b2b4d831524ea60ebaab4acf0772d361e8909a37195bed12f9)

Reply:
It should look something like this:

<img width="362" alt="imagen" src="https://user-images.githubusercontent.com/80283482/180241427-58c25751-2c70-4272-a30b-1ebdad106db9.png">

<img width="364" alt="imagen" src="https://user-images.githubusercontent.com/80283482/180241337-a81a701d-42ac-4fe6-b02f-171d142ef429.png">